pages/index.js

```diff
@@ -3,7 +3,7 @@ import Heading from "../components/Heading";
 export default function Home() {
   return (
     <main>
-      <Heading>🐠 Tropical Fish Capstone Template 🐠</Heading>
+      
```

Comment:
So your main page is empty? Regarding the user, this is the place where everything should start.

Reply:
you're right, but problem is that my start page must contain the login section. So shall I move my content to the pages/index , yet?

Reply:
You don't have a login section for now, do you? If not, don't plan for the future ;)